### DISCUSSION TITLE
Billing reporting storage

### DISCUSSION THREAD

**Discussion by adaptive** [community/community#38928] [Codespaces]
### Select Topic Area

Bug

### Body

Billing reporting storage usage, all my codespaces have been deleted.
All together couldn't exceed the 17Gb. Is it possible they are hanging somewhere after deletion?

<img width="798" alt="Screenshot 2022-11-12 at 12 27 13" src="https://user-images.githubusercontent.com/17833/201473998-214236ee-9852-4388-a410-fe6a91b53279.png">

<img width="892" alt="Screenshot 2022-11-12 at 12 33 06" src="https://user-images.githubusercontent.com/17833/201474134-779d62c8-f638-4c91-a008-ea4740c01aab.png">

**Comment by EndBug** [comment]
I had the exact same issue, and I think it's related to prebuilds: from my [usage report](https://docs.github.com/en/billing/managing-billing-for-github-codespaces/viewing-your-github-codespaces-usage#viewing-github-codespaces-usage-for-your-personal-account) I see that from the 9th of November they started billing storage usage for Codespaces Prebuilds. And I almost immediately capped it because I forgot I had some running 😅 

<img width="958" alt="image" src="https://user-images.githubusercontent.com/26386270/201531847-edc0d1c5-6e3c-40ea-8c04-2642c8fd52a6.png">

**Reply by adaptive** [reply]
I only had one prebuild of 2.5 Gbs. Started one codespace, for 30 minutes and the remaining 2.5Gb. I believe codespaces storage billing is badly coded.

**Comment by erikaellr** [comment]
Hi @adaptive, we're happy you're here! For billing questions like these that are specific to your account you'll need to [reach out to support directly](https://support.github.com/contact/account). We aren't able to share details related to billing within the public community so you'll need to contact our support team for more information.

**Reply by adaptive** [reply]
will do because with all the codespaces deleted, and delete prebuilds "There are no prebuilds configured for this repository" before the billing cycle they are still showing. billing keeps going...

**Comment by craiglpeters** [comment]
FYI if your project is open source and you use prebuilds, see https://github.co/prebuilds_for_oss
